### ISSUE TITLE
Customizing tab display beyond just label text.

### ISSUE THREAD

**Issue by marcodlk** [plotly/dash#247]
I love the dcc.Tabs component but I have recently run into a dead-end as far as its current capabilities go. My program manager requested that I change the labels of the tabs from text to icons (i.e. with font-awesome) in order to take our dashboard to the next level visually. However, this seems impossible at the moment.

**Feature Request**
Allow tabs to be customizable beyond just text given by the "label" strings. In particular, allow use of icons.

**Comment by gvwilson** [comment]
cc @AnnMarieW

**Comment by AnnMarieW** [comment]
Yes, this is a popular request - here a more recent post https://github.com/plotly/dash/issues/2456

In the meantime, it's possible to have components as tab labels in the dash-bootstrap-components or the dash-mantine-components library.

**Comment by gvwilson** [comment]
Thanks @AnnMarieW - given that, as part of our effort to tidy up our public repositories I'm going to close this one. @marcodlk if you'd still like changes, please add comments to #2456 and I'll try to get it prioritized. Thanks - @gvwilson
